lib/ParseSIL/ParseSIL.cpp

```diff
@@ -963,20 +963,35 @@ static bool parseSymbolicValue(SymbolicValue &value, SILParser &SP,
         }
       }
       // Handle decimal case.
-      double floatValue;
-      P.Tok.getText().getAsDouble(floatValue);
+      double tmpValue;
```

Comment:
Plz move this line right above its first use `P.Tok.getText().getAsDouble(tmpValue);`

Reply:
Done.